nodebuilder/p2p/p2p.go

```diff
@@ -194,24 +194,24 @@ func (m *module) PubSubPeers(_ context.Context, topic string) ([]peer.ID, error)
 //nolint:dupl
 type API struct {
 	Internal struct {
-		Info                 func(context.Context) (peer.AddrInfo, error)                         `perm:"admin"`
+		Info                 func(context.Context) (peer.AddrInfo, error)                         `perm:"public"`
```

Comment:
Please change all perms back to admin, this is a low-level module that isn't meant to be exposed so publicly.

Reply:
ok